example/main.tf

```diff
@@ -3,23 +3,16 @@ provider "aws" {
 }
 
 module "example_team_ec_cluster" {
-  source = "github.com/ministryofjustice/cloud-platform-terraform-elasticache-cluster?ref=1.0"
-
-  /*
-   * When using this module through the cloud-platform-environments, the
-   * following two variables are automatically supplied by the pipeline.
-   *
-   */
-  // cluster_name           = "cloud-platform-live-0"
-  // cluster_state_bucket   = "cloud-platform-cluster-state-bucket"
+  source = "github.com/ministryofjustice/cloud-platform-terraform-elasticache-cluster"
```

Comment:
This should point to `?ref=2.0` (which will be the version that we'll cut after this is merged).

Also, this block needs to be formatted.